src/db/schema/ratingEvent.ts

```diff
@@ -0,0 +1,35 @@
+import { index, integer, sqliteTable, text } from "drizzle-orm/sqlite-core";
+import { userTbl } from "./auth";
+import { seasonsTbl } from "./season";
+
+export const ratingEventTbl = sqliteTable(
+  "ratingEvent",
+  {
+    id: integer("id", { mode: "number" }).primaryKey({ autoIncrement: true }),
+    createdAt: integer("createdAt", { mode: "timestamp" })
+      .notNull()
+      .$defaultFn(() => new Date()),
+    seasonId: integer("seasonId")
+      .notNull()
+      .references(() => seasonsTbl.id),
+    playerId: integer("playerId")
+      .notNull()
+      .references(() => userTbl.id),
+    data: text("data", { mode: "json" }).notNull(),
+    type: text("type", {
+      enum: ["quest"],
```

Comment:
Min tanke med `type` var at have dem til at være super snævre, fx `quest:win_3_in_a_row`, `quest:play_1v1` el. `quest:win_over_player` og at det så kun var nødvendigt at bruge `text` i mere avancerede tilfælde. Fx hvis vi gerne vil gemme hvem man har vundet over ved `quest:win_over_player`.

Reply:
Jeg tænkte på at gøre typen til at pege direkte på en `QuestType` fra det andet pr, men med den nuværende quest model, så ville typen være f.eks. `WinCount` hvor det så kunne være 3 gange, 5 gange eller 7 gange f.eks. som jo ville være en del af "quest" staten, spørgsmålet er om man har lyst til at kunne se forskel på dem

Reply:
Ved ikke hvor meget værdi jeg synes det giver som det ser ud nu, så den kunne måske bare fjernes 🤷 

Tror det er low [LOGAF](https://blog.danlew.net/2020/04/15/the-logaf-scale/) fra min side, så gør hvad du føler for 👌

Reply:
Mener du `type`? eller hele rating event modellen :D